src/null-safety/index.md

```diff
@@ -43,266 +49,6 @@ Dart null safety support is based on the following three core design principles:
   and its dependencies to null safety, you reap the full benefits of soundness
   —- not only fewer bugs, but smaller binaries and faster execution.
 
-## A tour of the null safety feature
```

Comment:
I wonder if the page is a bit hard to consume for a new developer now that it has zero code examples on it? I think it's great to move the majority of this elsewhere, but would it make sense to include just a bit of code to exemplify what null safety is?

A concrete suggestion: Just before the `## Null safety principles` section, include just the piece from the current tour about declaring variables that are null and non-null (that is the first two code snippets in that section), and then end that with "For an example-driven summary...." changed to "For further examples...". WDYT?

Reply:
I swapped it around a bit... ptal